### ISSUE TITLE
Can anyone please share checkpoints that we get after we complete both stages of training

### ISSUE THREAD

**Issue by tanishbajaj101** [yl4579/StyleTTS2#268]
I am running out of memory when I train on LJSpeech dataset. I want to finetune upon LJSpeech itself but I'm failing in the training stage itself as either I would run out memory, or would have to significantly restrict my max_len implying I wouldn't be able to generate longer duration clips.
So can anyone please share their checkpoints made on training stage, so I can finetune on a relatively smaller dataset.
Also, will I be able to succesfully finetune on checkpoints I receive from internet? (i'm new to this)

**Comment by Tristan-Hopkins** [comment]
https://huggingface.co/yl4579/StyleTTS2-LJSpeech/tree/main
https://huggingface.co/yl4579/StyleTTS2-LibriTTS/tree/main
You Can use their models for finetuning. I this what you were looking for?

**Comment by tanishbajaj101** [comment]
> https://huggingface.co/yl4579/StyleTTS2-LJSpeech/tree/main https://huggingface.co/yl4579/StyleTTS2-LibriTTS/tree/main You Can use their models for finetuning. I this what you were looking for?

yes, thanks a lot! can you tell me which of these does the StyleTTS2 uses by default? I explored it using the WebUI and python package prior to building myself

**Comment by 78Alpha** [comment]
LibriTTS should be the default. Although LJ should be better.

**Comment by martinambrus** [comment]
@tanishbajaj101 could you please close the issue if it's resolved to clear things up please?
